### PR TITLE
Normalizing filenames to characters that will be accepted by AWS

### PR DESCRIPTION
**Pull request by saramaebee**
Workaround for ANE-682.

### CODE REVIEW

File: python/scripts/json-conversion/fossa-deps-conversion.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def saveFossaDepsJson(dictionary):
     file.close()
 
 def findRpmTarVendoredDependencies():
+    reg = re.compile('[a-zA-Z0-9\.-_]')
```
Comment (cmboling) [inline python/scripts/json-conversion/fossa-deps-conversion.py:23]: Need to import re at the beginning of the script.

Comment (saramaebee) [reply python/scripts/json-conversion/fossa-deps-conversion.py:24]: my bad!

File: python/scripts/json-conversion/fossa-deps-conversion.py [hunk 2]
```diff
@@ -38,7 +40,7 @@ def findRpmTarVendoredDependencies():
```
Comment (cmboling) [inline python/scripts/json-conversion/fossa-deps-conversion.py:40]:
```suggestion
normalised_filname =  "".join(reg.findall(filename)
```

File: python/scripts/json-conversion/fossa-deps-conversion.py [hunk 3]
```diff
@@ -38,7 +40,7 @@ def findRpmTarVendoredDependencies():
 
             # take out dupes
             if not any(dep.get('name', None) == filename for dep in vendoredDeps["vendored-dependencies"]):
-                vendoredDeps["vendored-dependencies"].append({"name": filename, "path":path})
+                vendoredDeps["vendored-dependencies"].append({"name": "".join(reg.findall(filename)), "path":path})
```
Comment (cmboling) [inline python/scripts/json-conversion/fossa-deps-conversion.py:43]:
```suggestion
                vendoredDeps["vendored-dependencies"].append({"name": normalised_filname, "path":path})
```

**Comment by saramaebee** [comment]
ok i think it all looks good, @cmboling we good to merge?

**Comment by cmboling** [comment]
LGTM

**Pull request merged**